src/components/MultiHopTrade/components/TradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -50,7 +50,7 @@ import {
   selectHopSellAccountId,
   selectTradeSlippagePercentageDecimal,
 } from '@/state/slices/tradeQuoteSlice/selectors'
-import { tradeQuoteSlice } from '@/state/slices/tradeQuoteSlice/tradeQuoteSlice'
+import { tradeQuote as tradeQuoteSlice } from '@/state/slices/tradeQuoteSlice/tradeQuoteSlice'
```

Comment:
`tradeQuote` already exists in the scope we consume this, so we need to alias this here